scripts/scil_screenshot_bundle.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import argparse
+import os
+
+from dipy.io.stateful_tractogram import Space, StatefulTractogram
+from dipy.io.streamline import load_tractogram
+from dipy.tracking.streamline import transform_streamlines
+from fury import actor
+import matplotlib.pyplot as plt
+import nibabel as nib
+import numpy as np
+from scipy.ndimage import map_coordinates
+
+from scilpy.io.utils import (add_overwrite_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist)
+from scilpy.utils.image import register_image
+from scilpy.viz.screenshot import display_slices
+
+DESCRIPTION = """
+   Register bundle to a template for screenshots using a reference.
+   The template are in /mnt/braindata/Other/simple_template_viz/
+   Axial, coronal and sagittal slices are captured.
+   Sagittal can be capture from the left (default) or the right.
+   """
+
+
+def _build_args_parser():
+    p = argparse.ArgumentParser(
+        description=DESCRIPTION,
+        formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('in_bundle',
+                   help='Path of the input bundle')
+    p.add_argument('in_anat',
```

Comment:
Shouldn't this be added via add_reference ?

Reply:
This one is mandatory for the script, and serve as a reference for registration (first purpose)